utilities/test_suite/HIP/runTests.py

```diff
@@ -181,7 +193,7 @@ def rpp_test_suite_parser_and_validator():
             print("Functionalities - ", funcCount)
 
             # Print summary of log
-            print("\n\nFunctionality\t\t\t\t\t\tFrames Count\tmax(ms)\t\tmin(ms)\t\tavg(ms)\n")
+            print("\n\nFunctionality\t\t\t\t\t\tFrames Count\tmax(ms/batch)\t\tmin(ms/batch)\t\tavg(ms/batch)\n")
```

Comment:
Please check this line. These \t\ts are meant to be tabs, but I was seeing "\t\t.." printed out in the output display during the demo. Also, where are we displaying the batch size and the max image size in the batch?